libdnf/libdnf.pc.in

```diff
@@ -3,8 +3,8 @@ libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/include
 
 Name: libdnf
-Description: Simple package manager interface and librepo
+Description: Simple package manager interface using libsolv and librepo
 Version: @LIBDNF_VERSION@
-Requires: glib-2.0, gobject-2.0, librepo, rpm
+Requires: glib-2.0, gobject-2.0, librepo, libsolv, libsolvext, rpm
```

Comment:
A major part of the idea of libdnf is that its public interface does *not* expose libsolv.  So this should be `Requires.private`.  And so in fact should `librepo` and probably `rpm`.

Reply:
At this point it does...

Reply:
When they work as private interfaces, I'd do that. I found out about this because it doesn't.

Reply:
@cgwalters I moved the libsolv Requires to `Requires.private`. It seems that we don't directly expose it in git master, from what I can tell...